module/Finna/src/Finna/Db/Row/FinnaUserTrait.php

```diff
@@ -293,6 +293,26 @@ public function getDisplayName()
         return $this->getUsername();
     }
 
+    /**
+     * Get a displayable version of username
+     *
+     * @return string
+     */
+    public function getDisplayableUsername(): string
+    {
+        if (strpos($this->username, ':')) {
+            [$view, $username] = explode(':', $this->username);
```

Comment:
Rajaa varmuuden vuoksi se, miten moneen osaan explodetaan:
```suggestion
            [$view, $username] = explode(':', $this->username, 2);
```